tools/page-name-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Do you think we can reuse https://github.com/alphagov/govuk-developer-docs/blob/master/spec/pages/manual_page_spec.rb for this?

Reply:
Yup! I'll do that.